stable/spartakus/README.md

```diff
@@ -0,0 +1,67 @@
+# Spartakus
+
+[Spartakus](https://github.com/kubernetes-incubator/spartakus) aims to collect information about Kubernetes clusters. This information will help the Kubernetes development team to understand what people are doing with it, and to make it a better product.
+
+## TL;DR;
+
+```console
+$ helm install stable/spartakus
```

Comment:
It meets the requirements as far as I can tell. No need to artificially incubate it if its ready to go now.